packages/tasks/src/model-libraries.ts

```diff
@@ -116,6 +116,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path_extension:"pth"`,
 		snippets: snippets.audioseal,
 	},
+	bagel: {
```

Comment:
```suggestion
	"bagel-mot": {
```

Added as a suggestion. You'd also need to modify the [library name in the model card](https://huggingface.co/ByteDance-Seed/BAGEL-7B-MoT/blob/main/README.md?code=true#L6).